node/tests/fraud_test.go

```diff
@@ -73,3 +73,63 @@ func TestFraudProofBroadcasting(t *testing.T) {
 	require.NoError(t, err)
 	require.NotNil(t, proofs)
 }
+
+/*
+ Test-Case: Light node receives Fraud Proof using Fraud Sync
+ Pre-Requisites:
+ - CoreClient is started by swamp.
+ Steps:
+ 1. Create a Bridge Node(BN) with broken extended header at height 10.
+ 2. Start a BN.
+ 3. Create a Full Node(FN) with a connection to BN as a trusted peer.
+ 4. Start a FN.
+ 5. Subscribe to BEFP and wait when it will be received.
```

Comment:
we have changed the adr from `BEFP` to fraud. 
Should we change the namings here to be more compliant?